accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/HtmlWriter.java

```diff
@@ -0,0 +1,80 @@
+package biz.netcentric.cq.tools.actool.ui;
+
+import static org.apache.commons.lang3.StringEscapeUtils.escapeHtml4;
+
+import java.io.PrintWriter;
+
+/** Utility class for writing html as regular templating cannot be used for web console plugin. */
+class HtmlWriter {
+
+    final PrintWriter pw;
+    final boolean isTouchUi;
+
+    HtmlWriter(final PrintWriter pw, boolean isTouchUi) {
+        this.pw = pw;
+        this.isTouchUi = isTouchUi;
+    }
+
+    void openTable(String id) {
+        pw.println("<table id='"+id+"' " + (isTouchUi ? " is='coral-table'" : " class='content' cellpadding='0' cellspacing='0' width='100%'") + ">");
```

Comment:
rather use fluent api via `append` instead of all the concatenation. Easier to read and should be faster